### PR TITLE
Document AWS synchronous behavior and performance considerations

### PR DESCRIPTION
**Pull request by oleiade**
This PR intends to add Blockquote to the jslib-aws pages documenting the blocking behavior of some of its APIs, its potential impact on performance and results, and how to get around that.

Screenshot 2023-04-24 at 14 45 34

⚠️ I have included the change only for a single client now. Once we agree on the proper phrasing and setup, I will update the rest of the pages 👍🏻

### CODE REVIEW

**Review by MattDodsonEnglish — COMMENTED** [review]
LGTM with some minor suggestions.

You might already be planning to, but consider using the shared component so that you can single source the file. For examples, checkout the [`experimental-blockquote` source](https://github.com/grafana/k6-docs/tree/main/src/components/shared/experimental-blockquote).

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/07 SQSClient.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ description: 'SQSClient enables interaction with the AWS Simple Queue Service (S
 excerpt: 'SQSClient allows interacting with the AWS Simple Queue Service (SQS)'
 ---
 
-`SQSClient` interacts with the AWS Simple Queue Service (SQS). With it, the user can send messages to specified queues and list available queues in the current region. `SQSClient` operations are blocking. k6 recommends reserving their use for the [`setup`](/using-k6/test-lifecycle/) and [`teardown`](/using-k6/test-lifecycle/) stages as much as possible.
+<Blockquote mod="Attention" title="Performance considerations and best practices">
```
[the remark this answers is not in the document]
Comment (oleiade) [reply src/data/markdown/docs/20 jslib/01 jslib/01 aws/07 SQSClient.md:8]: Very good point 🙇🏻

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/07 SQSClient.md [hunk 2]
```diff
@@ -5,7 +5,13 @@ description: 'SQSClient enables interaction with the AWS Simple Queue Service (S
...
+The APIs provided by this jslib operate synchronously, which means k6 must wait for operations using the client classes to finish before proceeding with the rest of the script. In some cases, this could affect performance and test results, for example, when downloading large files from S3.
+
+To minimize the impact on test performance, we recommend using these operations in the [`setup`](/using-k6/test-lifecycle/) and [`teardown`](/using-k6/test-lifecycle/) functions, which won't influence the test results as they occur before and after the test run.
```
[the remark this answers is not in the document]
Comment (oleiade) [reply src/data/markdown/docs/20 jslib/01 jslib/01 aws/07 SQSClient.md:11]: That's a really fair point 👏🏻

**Comment by oleiade** [comment]
Note that once the [jslib supports `async/await`](https://github.com/grafana/k6-jslib-aws/issues/41). We will drop this blockquote from the client pages.

**Review by MattDodsonEnglish — APPROVED** [review]
Looks like the components renders just fine to me. Who knows? Maybe this is the start of you and Gatsby becoming BFFs.

welcome Theo

**Pull request merged**